driver/main.c

```diff
@@ -1523,13 +1523,14 @@ static inline int drop_event(struct ppm_consumer_t *consumer,
 	}
 
 	if (consumer->dropping_mode) {
+		nanoseconds ns2 = ns;
```

Comment:
I'd move this declaration nearest as possible to its usage, if you are ok with that!

Reply:
In C code, there is a common constraint, that variables should be defined at the start of a block. Given that constraint, it is the closest it can be to the usage.

Reply:
Uh i was thinking about c99; you are right obviously. Thanks!